- set correct DYLD_LIBARY_PATH from build-system (instead of external
  from ebuild)
- don't probe for Fink and MacPorts libraries, we only want Gentoo
  Prefix' libs

--- parrot-2.3.0/config/gen/makefiles/root.in
+++ parrot-2.3.0/config/gen/makefiles/root.in
@@ -555,7 +555,7 @@
 
 # Libraries
 LIBPARROT_STATIC    = @blib_dir@/@libparrot_static@
-#IF(darwin):export DYLD_LIBRARY_PATH := @blib_dir@:$(DYLD_LIBRARY_PATH)
+#IF(darwin):export DYLD_LIBRARY_PATH := @build_dir@/@blib_dir@:$(DYLD_LIBRARY_PATH)
 #IF(win32):LIBPARROT_SHARED  = @libparrot_shared@
 #ELSE:LIBPARROT_SHARED  = @blib_dir@/@libparrot_shared@
 
--- parrot-2.3.0/config/init/hints/darwin.pm
+++ parrot-2.3.0/config/init/hints/darwin.pm
@@ -56,9 +56,6 @@
 
     $flagsref->{linkflags} .= " -undefined dynamic_lookup";
 
-    _probe_for_libraries($conf, $flagsref, 'fink');
-    _probe_for_libraries($conf, $flagsref, 'macports');
-
     for my $flag ( keys %$flagsref ) {
         $flagsref->{$flag} =~ s/^\s+//;
     }
